_pages/dev/context/automatic-context-configuration.md

```diff
@@ -54,7 +54,7 @@ When using PWA, the back end response that provides the base sites can be cached
     {
       "name": "basesites",
       "urls": [
-        "*/rest/v2/basesites?fields=baseSites\\(uid,defaultLanguage\\(isocode\\),urlEncodingAttributes,urlPatterns,stores\\(currencies\\(isocode\\),defaultCurrency\\(isocode\\),languages\\(isocode\\),defaultLanguage\\(isocode\\)\\)\\)*"
```

Comment:
FYI @WeizhengSap I relaxed the regex, by dropping the closing parenthesis of `fields=baseSites(....`, just to allow for new params we might add going on. For example `,theme`